src/main/java/com/spectralogic/ds3cli/Arguments.java

```diff
@@ -83,7 +83,7 @@
         directory.setArgName("directory");
         final Option objectName = new Option("o", true, "The name of the object to be retrieved or stored");
         objectName.setArgName("objectFileName");
-        final Option prefix = new Option("p", true, "Restores only objects who's names start with prefix.  Used with `get_bulk`");
+        final Option prefix = new Option("p", true, "Restores only objects who's names start with prefix.  Used with `get_bulk` and 'put_object'");
```

Comment:
We should probably add more here to explain that with bulk_put and put_object that it's used to add a prefix.